src/test/scala/dx/exec/WfFragRunnerTest.scala

```diff
@@ -26,25 +26,32 @@ class WfFragRunnerTest extends AnyFlatSpec with Matchers {
     DxInstanceType("mem_ssd_unicorn", 100, 100, 4, 1.00f, Vector(("Ubuntu", "16.04")), gpu = false)
   private val instanceTypeDB = InstanceTypeDB(pricingAvailable = true, Vector(unicornInstance))
 
-  private def setup(): (DxPathConfig, DxFileAccessProtocol) = {
+  private def setup(): (DxPathConfig, FileSourceResolver) = {
     // Create a clean directory in "/tmp" for the task to use
-    val jobHomeDir: Path = Paths.get("/tmp/dxwdl_applet_test")
-    SysUtils.deleteRecursive(jobHomeDir.toFile)
-    SysUtils.safeMkdir(jobHomeDir)
+    val jobHomeDir: Path = Files.createTempDirectory("dxwdl_applet_test")
```

Comment:
In the next PR, in [TaskRunnerTest](https://github.com/dnanexus/dxWDL/pull/430/files#diff-001802157ca7e83cd5725af52b1e28a6R138) the deleteRecursive() and createDirectories() methods are called. Why they are not called here?

Reply:
createTempDir creates a new temp directory - there is no need to delete it and recreate it as there will never be a collision. I'll make sure it's fixed in TaskRunnerTest too.